cg/apps/hermes/models.py

```diff
@@ -16,23 +17,22 @@ class CGTag(BaseModel):
     mandatory: Optional[bool] = False
 
 
+def check_mandatory_exists(file: CGTag) -> Optional[CGTag]:
+    if file.mandatory:
+        assert Path(file.path).exists(), f"Mandatory file cannot be found at {file.path}"
+        return file
+    if not Path(file.path).exists():
+        LOG.info(f"Optional file {file.path} not found, removing from bundle")
+        return
+    return file
+
+
+def remove_invalid(files: List[Optional[CGTag]]) -> List[CGTag]:
+    return [file for file in files if file]
+
 class CGDeliverables(BaseModel):
     """Class that specifies the output from hermes"""
 
     pipeline: str
     bundle_id: str
-    files: List[CGTag]
-
-    @validator("files", each_item=True)
-    def check_mandatory_exists(cls, file):
-        if file.mandatory:
-            assert Path(file.path).exists(), f"Mandatory file cannot be found at {file.path}"
-            return file
-        if not Path(file.path).exists():
-            LOG.info("Optional file %s not found, removing from bundle", file.path)
-            return
-        return file
-
-    @validator("files")
-    def remove_invalid(cls, value):
-        return [item for item in value if item]
```

Comment:
`each_item` is no longer supported in Pydantic V2, see [migration guide](https://docs.pydantic.dev/dev-v2/migration/#changes-to-validators). Instead, one is supposed to annotate the type like `List[Annotated[int, Field(ge=0)]]`. See [examples](https://docs.pydantic.dev/dev-v2/usage/validators/#generic-validated-collections).